source/LibUISharp/src/Assembly.cs

```diff
@@ -27,4 +27,5 @@
 [assembly: SuppressMessage("Style", "IDE1006:Naming Styles", Scope = "type", Target = "~T:LibUISharp.Internal.LibuiLibrary.uiTextWeight")]
 [assembly: SuppressMessage("Style", "IDE1006:Naming Styles", Scope = "type", Target = "~T:LibUISharp.Internal.LibuiLibrary.uiUnderline")]
 [assembly: SuppressMessage("Style", "IDE1006:Naming Styles", Scope = "type", Target = "~T:LibUISharp.Internal.LibuiLibrary.uiUnderlineColor")]
-[assembly: SuppressMessage("Style", "IDE1006:Naming Styles", Scope = "type", Target = "~T:LibUISharp.Internal.LibuiLibrary.uiWindowResizeEdge")]
\ No newline at end of file
+[assembly: SuppressMessage("Style", "IDE1006:Naming Styles", Scope = "type", Target = "~T:LibUISharp.Internal.LibuiLibrary.uiWindowResizeEdge")]
+[assembly: System.CLSCompliant(true)]
```

Comment:
I'd prefer to keep all `SupressMessageAttributes` on the bottom, as they won't affect the build.